src/main/java/de/tum/in/www1/artemis/domain/TextExercise.java

```diff
@@ -19,6 +19,13 @@
     @Lob
     private String sampleSolution;
 
+    @Column(name = "automatic_text_assessment")
```

Comment:
Could we change this to an enum AssessmentType and allow in general MANUAL, SEMI_AUTOMATIC and AUTOMATIC and move this to Exercise.java?

Then, we can also specify this in other exercises, e.g. modeling in the future. And we need to allow SEMI_AUTOMATIC (i.e. automatic + manual) for programming exercises in PGdP as well, so it would be great if we could reuse this enum then.